easybuild/easyconfigs/p/pyEGA3/pyEGA3-4.0.0-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'PythonBundle'
+
+name = 'pyEGA3'
+version = '4.0.0'
+
+homepage = 'https://github.com/EGA-archive/ega-download-client'
+description = """ A basic Python-based EGA download client """
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+builddependencies = [('binutils', '2.37')]
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('tqdm', '4.62.3'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('humanize', version, {
```

Comment:
the `humanize` version has nothing to do with the `pyEGA3` version, so decouple those, use `'4.0.0'` here